shared/routes/RouteInformationTypes.ts

```diff
@@ -14,7 +14,7 @@ import {
   SearchRouteType,
   SprungbrettOfferRouteType,
   TuNewsType,
-} from '.'
+} from './index'
```

Comment:
You don't need to add the `index` everywhere. An import from a directory automatically points to the `index` file, so `'.'` leads to exactly the same file as `'./index'`